src/script/finale.ts

```diff
@@ -4,9 +4,16 @@ import { HelpProvider } from "./help/helpProvider";
 import { Model } from "./modelParser";
 import { PathHelper } from "./pathHelper";
 import { ViewFactory } from "./viewFactory";
+import { isNullOrUndefined } from "util";
+import { Main } from "./main";
 
 document.addEventListener("DOMContentLoaded", () => {
-    generateDocument(document.location.search);
+    if (isNullOrUndefined(sessionStorage.getItem(document.location.search))) {
+        generateDocument(document.location.search);
+        } else {
+            Main.viModel = JSON.parse(sessionStorage.getItem(document.location.search));
+            LoadDOM(document.location.search);
```

Comment:
You could consider calling `LoadDOM()` `GenerateViewForModel()` or `GenerateDOMForModel()`. Since we're already in the window's "DOMContentLoaded", `LoadDOM()` seems a little misleading.

Reply:
Okay I will change it.